crates/types/service-protocol/dev/restate/service/protocol.proto

```diff
@@ -317,6 +318,9 @@ message CallEntryMessage {
   // If this invocation has a key associated (e.g. for objects and workflows), then this key is filled in. Empty otherwise.
   string key = 5;
 
+  // If present, it must be non empty.
+  optional string idempotency_key = 6;
```

Comment:
This means that an empty string "" is an invalid `idempotency_key`?

Reply:
yes.